### PR TITLE
Sample sentences

### PR DESCRIPTION
**Pull request by ak3ra**
Added sample sentences in UI. 

TODO: 
- Make sentences selectable into the textarea

### CODE REVIEW

**Comment by ak3ra** [comment]
@NLSanyu  I have added functionality to automatically paste the sample texts to text area for translation

**Comment by NLSanyu** [comment]
> @NLSanyu I have added functionality to automatically paste the sample texts to text area for translation

Okay, I'll review today

File: js/scripts.js [hunk 1]
```diff
@@ -177,3 +177,60 @@ negFeedback.addEventListener(
...
+    if (content.style.maxHeight){
+      content.style.maxHeight = null;
+    } else {
+      content.style.maxHeight = content.scrollHeight + "px";
+    }
+  });
+}
+
+// Tab lists
+function openCity(evt, cityName) {
```
Comment (NLSanyu) [inline js/scripts.js]: What does "city" mean in this context? I see that the function name and variable "city" included

**Review by NLSanyu — CHANGES_REQUESTED** [review]
Great job @ak3ra. It looks good and works well. However, please take note of the following things:

* I have noticed that when a category of sample sentences (common phrases) is chosen, that category's name appears with a dash after it, as shown in the screenshot below for the category "**Socials**". Is that on purpose?

Screenshot 2022-02-02 at 09 24 29

* I have also left a comment about an issue I found in the code as part of this review, please check it out.

* **Note**: After working on the feedback, please format the HTML, CSS and JS files you updated using default formatting or using a linter (Prettier, for example). This is to ensure that we have good code quality.

**Comment by NLSanyu** [comment]
@ak3ra I see that you fixed the naming issue in the code as requested, thanks.
What about the issue with the dashes appearing on a selected sample sentence category?

**Review by NLSanyu — APPROVED** [review]

**Pull request merged**